<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>demo</class>
 <widget class="QDialog" name="demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SlidingStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="p1_container" native="true">
         <widget class="QLabel" name="p1_title">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>171</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 24px;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Welcome!</string>
          </property>
         </widget>
         <widget class="QLabel" name="p1_desc">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>431</width>
            <height>151</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 14px;</string>
          </property>
          <property name="text">
           <string>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="p2_container" native="true">
         <widget class="QLabel" name="p2_title">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>301</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 24px;
font-weight: bold;
</string>
          </property>
          <property name="text">
           <string>Widgets</string>
          </property>
         </widget>
         <widget class="QTabWidget" name="tabWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>431</width>
            <height>131</height>
           </rect>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
           <widget class="QRadioButton" name="radioButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>121</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Radio Button</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>131</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Radio Button 2</string>
            </property>
           </widget>
           <widget class="QComboBox" name="comboBox">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>70</y>
              <width>141</width>
              <height>23</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>Item 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Item 2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Item 3</string>
             </property>
            </item>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="SlidingStackedWidget" name="stackedWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Nested StackWidget</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Page 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is nested Stack widget. i.e, StackWidget inside a Stack Widget.&lt;/p&gt;&lt;p&gt;The Class&lt;span style=&quot; font-weight:600;&quot;&gt; can handle nested StackWidget, &lt;/span&gt;the support was added by &lt;span style=&quot; font-weight:600;&quot;&gt;Keshav Bhatt&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nestedGoToPage2">
             <property name="text">
              <string>Go to page 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>24</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Nested StackWidget</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Page 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is nested Stack widget. i.e, StackWidget inside a Stack Widget.&lt;/p&gt;&lt;p&gt;The Class&lt;span style=&quot; font-weight:600;&quot;&gt; can handle nested StackWidget, &lt;/span&gt;the support was added by &lt;span style=&quot; font-weight:600;&quot;&gt;Keshav Bhatt&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nestedGoToPage1">
             <property name="text">
              <string>Go back</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_5">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="p4_container" native="true">
         <widget class="QLabel" name="p4_title">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>271</width>
            <height>61</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 24px;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Last page</string>
          </property>
         </widget>
         <widget class="QLabel" name="p4_desc">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>431</width>
            <height>141</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 14px;</string>
          </property>
          <property name="text">
           <string>Est ridiculus Class consectetuer leo nullam. Convallis porta condimentum dapibus. Suspendisse aliquam ligula elementum blandit. Massa sit. Tempor nonummy commodo odio id, bibendum.

Consequat, cras suspendisse molestie, cras purus integer tristique quam est lorem.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="prev">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Prev</string>
       </property>
       <property name="icon">
        <iconset theme="arrow-left">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="next">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset theme="arrow-right">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SlidingStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>slidingstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
